week3/homework/src/api.js

```diff
@@ -0,0 +1,91 @@
+const uuid = require('uuid');
+const Util = require('./util');
+const FILENAME = 'todos.json';
+const util = new Util(FILENAME);
+
+class Api {
+  async readTodos(req, res) {
+    let todos = await util.read();
+    res.send(todos);
+  }
+
+  async createTodo(req, res) {
+    try {
+      const todos = await util.read();
+      const todo = {
+        description: req.body.todo.description,
+        id: uuid.v4(),
+        done: false,
+      };
+      todos.push(todo);
+      await util.save(todos);
+      res.end('todo was successful created!');
+    } catch (err) {
+      res.end(err);
```

Comment:
This could send some information no need for the user to have any knowledge about
perhaps it's best to have a hard coded message instead as: `{ error: { type: "server error" , description: "Oops! something went wrong." } }`

and don't forgot the status code.